### DISCUSSION TITLE
How do I use my own VPN files

### DISCUSSION THREAD

**Discussion by deltamelter** [haugene/docker-transmission-openvpn#2367] [Q&A] [answered]
How do I use my own VPN files, smartdnsproxy openvpn files that this docker downloads at init are broken.
I created a ticket to have them updated, but probably did it wrong so it is ignored, so asking here...   How can I help myself?

**Comment by ilike2burnthing** [comment, marked as answer]
https://haugene.github.io/docker-transmission-openvpn/supported-providers/#using_a_local_single_ovpn_file_from_provider
